cidc_api/shared/gcloud_client.py

```diff
@@ -149,92 +134,73 @@ def revoke_upload_access(user_email: str):
     bucket.set_iam_policy(policy)
 
 
-def _build_intake_prefix(
-    user_id: int, user_email: str, trial_id: str, upload_type: str
-) -> str:
-    usernameish = user_email.split("@")[0].replace(".", "")
-    return (
-        f"{_build_trial_upload_prefix(trial_id, upload_type)}/{usernameish}-{user_id}"
-    )
-
-
-def grant_intake_access(
-    user_id: int, user_email: str, trial_id: str, upload_type: str
-) -> str:
+def get_intake_bucket_name(user_email: str) -> str:
     """
-    Give a user upload access to a subdirectory of the intake bucket with prefix like
-    <trial id>/<upload type>/<user email prefix>-<user id>
-
-    Return the GCS URI to which access has been granted.
+    Get the name for an intake bucket associated with the given user.
+    Bucket names will have a structure like <intake
     """
-    prefix = _build_intake_prefix(user_id, user_email, trial_id, upload_type)
-
-    logger.info(f"Granting intake access on {prefix} to {user_email}")
-
-    return grant_conditional_gcs_access(
-        GOOGLE_INTAKE_BUCKET, prefix, GOOGLE_INTAKE_ROLE, user_email
-    )
+    # 10 characters should be plenty, given that we only expect
+    # a handful of unique data uploaders - we get 16^10 possible hashes.
+    email_hash = hashlib.sha1(bytes(user_email, "utf-8")).hexdigest()[:10]
```

Comment:
nit: no salt though?

Reply:
Per our offline discussions, I'm not going to add a salt, since the main goal of hashing emails is to avoid directly publishing user information in bucket URLs, not to obscure which bucket belongs to which user.